plugins/module_utils/imagebuilder_module.py

```
@@ -0,0 +1,220 @@
+    # --- ImageBuild operations ---
+
+    def get_image_build(self, name: str, with_exports: Optional[bool] = None) -> Optional[Any]:
+        try:
+            kwargs = {}
+            if with_exports is not None:
+                kwargs['with_exports'] = with_exports
+            return self.call_api(self.imagebuild_api.get_image_build, name, **kwargs)
+        except NotFoundException:
+            return None
+        except ApiException as e:
+            raise FlightctlApiException(f"Unable to get ImageBuild {name}: {e}")
```

Ruff (0.15.9)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)